nucypher/characters/lawful.py

```diff
@@ -793,6 +794,9 @@ def __init__(self,
                     suspicious_activity_tracker=self.suspicious_activities_witnessed,
                     serving_domains=domains,
                 )
+                # attach status app to rest_app
+                UrsulaStatusPage(self, self.nickname, rest_app, '/status/')
```

Comment:
This could be improved i.e. added within the `make_rest_app` call by David's PR #961 since I needed an instance of Ursula to pass in.